_packaging/gaphor.spec

```diff
@@ -70,14 +87,27 @@ a = Analysis(
         },
     },
     hookspath=["."],
-    runtime_hooks=[],
+    runtime_hooks=[f"use_gtk_{os.getenv('GAPHOR_PKG_GTK', '4')}.py"],
     excludes=[],
     win_no_prefer_redirects=False,
     win_private_assemblies=False,
     cipher=block_cipher,
     noarchive=False,
 )
+
 pyz = PYZ(a.pure, a.zipped_data, cipher=block_cipher)
+
+pyinstaller_versionfile.create_versionfile(
+    output_file="windows/file_version_info.txt",
+    version=get_semver_version(),
+    company_name="Gaphor",
+    file_description="Gaphor",
+    internal_name="Gaphor",
+    legal_copyright="Copyright © 2001-2021 Arjan J. Molenaar and Dan Yeaw.",
```

Comment:
While we are at it, what do you think about importing datetime and then:

`legal_copyright=f"Copyright 2011-{datetime.date.today().year}...`

Reply:
I want to make one copyright variable: we also need it for the macOS build.

Reply:
Great, I like it!